linux_os/guide/services/ssh/file_owner_sshd_config/rule.yml

```
@@ -12,6 +14,11 @@ rationale: |-
 
 severity: medium
 
+identifiers:
+    cce@rhel6: 82900-2
+    cce@rhel7: 82899-6
+    cce@rhel8: 82998-8
```

```suggestion
    cce@rhel8: 82898-8
```
Invalid CCE, just a typo.